src/app/(spaces)/PublicSpace.tsx

```
@@ -582,8 +287,9 @@ export default function PublicSpace({
             await loadEditableSpaces(); // First load
             await loadSpaceTab(newSpaceId, spacePageData.defaultTab);
 
-            // Load remaining tabs
-            const tabOrder = localSpaces[newSpaceId]?.order || [];
+            // Load remaining tabs - get fresh reference after loading operations
+            const freshLocalSpaces = useAppStore((state) => state.space.localSpaces);
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)